Controllers/RedirectController.cs

```diff
@@ -37,6 +41,11 @@ public async Task<IActionResult> Index(string contentItemId)
 
             var part = contentItem.As<RedirectPart>();
 
+            if (part.ToUrl.StartsWith("/", System.StringComparison.Ordinal))
+            {
+                return new RedirectResult($"{_tenantService.GetTenantUrl()}{part.ToUrl.TrimStart('/')}", part.IsPermanent);
```

Comment:
The `GetTenantUrl` returns a URL without a trailing `/` and any prefixed `/` is being stripped so this causes the redirect URL to be incorrect. For example, if my tenant URL prefix was "example" and I was redirecting to "/search", the redirect URL would become "/examplesearch" instead of "/example/search".